src/components/Toolbar/Toolbar.js

```diff
@@ -215,51 +216,49 @@ function Toolbar(props) {
             sx={{
               width: '100%',
               display: 'flex',
-              flexDirection: 'column'
+              flexDirection: 'column',
+              textTransform: 'none'
             }}
           >
             <ResourceIcon />
-            <Typography style={{ textTransform: 'none' }} fontSize={'small'}>
-              Resources
-            </Typography>
+            <Typography fontSize={'small'}>Resources</Typography>
```

Comment:
Curly braces are not necessary for the `fontSize` prop:
```suggestion
            <Typography fontSize="small">Resources</Typography>
```